elide-async/src/main/java/com/yahoo/elide/async/operation/AsyncAPIUpdateOperation.java

```diff
@@ -40,10 +40,10 @@ public void run() {
             asyncAPIDao.updateAsyncAPIResult(queryResultObj, queryObj.getId(), queryObj.getClass());
 
         } catch (InterruptedException e) {
-            log.error("InterruptedException: {}", e);
+            log.error("InterruptedException: {}", e.toString());
```

Comment:
If the stack trace is intended, we need to instead remove the `{}` from the formatting
```java
log.error("InterruptedException", e);
```

Reply:
stacktrace was not intended.

Reply:
What is the purpose of:
 Thread.currentThread().interrupt();
?